generative/maisi/maisi_inference_tutorial.ipynb

```diff
@@ -0,0 +1,288 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    !python -c "import os; print(os.path.isfile('models/autoencoder_epoch273.pt'))" || python -m monai.bundle download_large_files --bundle_path .
</pre><p>The expected outcome of this line seems to be</p><ul><li>Downloading the ckpt if it is not present</li></ul><p>However it will only print <code>False</code> without downloading files.</p><p>A simple fix can be</p>

<pre class="ql-syntax" spellcheck="false">!python -c "import os; assert os.path.isfile('models/autoencoder_epoch273.pt')" || python -m monai.bundle download_large_files --bundle_path .
</pre><p> </p><p>It would be better to have a Data Preparation section, which checks every single file in the <code>large_files.yaml</code>, and follow other tutorials by using <code>MONAI_DATA_DIRECTORY</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1741/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Agreed, could you please remove the download part here to "setup data directory" and "download dataset".</p><p>Please refer to: <a href="https://github.com/Project-MONAI/tutorials/blob/main/3d_segmentation/spleen_segmentation_3d.ipynb" target="_blank">https://github.com/Project-MONAI/tutorials/blob/main/3d_segmentation/spleen_segmentation_3d.ipynb</a></p><p> </p>

Reply:
Thanks, I just added MONAI_DATA_DIRECTORY. But I cannot finalize "Download dataset and trained model weights" before the link is finalized.

Reply:
I adjusted that block to only put `datasets` in the `MONAI_DATA_DIRECTORY`